include/force_torque_sensor.h

```diff
@@ -0,0 +1,93 @@
+#ifndef _FORCE_TORQUE_SENSOR_H_
+#define _FORCE_TORQUE_SENSOR_H_
+
+#include <gazebo/gazebo.hh>
+#include <gazebo/physics/physics.hh>
+#include <gazebo/common/common.hh>
+#include <XBotCoreModel.h>
+#include <rtt/Port.hpp>
+#include <rst-rt/dynamics/Wrench.hpp>
+#include <gazebo-7/gazebo/sensors/ForceTorqueSensor.hh>
```

Comment:
Change ```#include <gazebo-7/gazebo/sensors/ForceTorqueSensor.hh>```
into this ```#include <gazebo/sensors/ForceTorqueSensor.hh>```

Because if we hardcode gazebo-7, then we will have problems once we change to another gazebo version in the future.